### ISSUE TITLE
sbb-link: check alternative accessibility approach

### ISSUE THREAD

**Issue by jeripeierSBB** [lyne-design-system/lyne-components#2376]
Evaluate the following method, especially in terms of SEO usage:
In our link we remove the content of <a> element and made it a sibling. We set aria-hidden onto the a element which has no content anymore and we span this with absolute positioning on top of all the real content.
With that it's completely excluded from accessibility and therefore doesn't read "link link" anymore.
We don't know the influence on SEO when there is a hidden <a> element with no content. Need to be investigated.

Something like: 
```
    return html`
      <a aria-hidden="true" class="sbb-inset" ${spread(attributes)}></a>
      <div class="sbb-link">
        ${
          this.variant !== 'inline'
            ? html`<span class="sbb-link__icon">
                <slot name="icon">
                  ${this.iconName ? html`<sbb-icon name=${this.iconName}></sbb-icon>` : nothing}
                </slot>
              </span>`
            : nothing
        }
        <slot></slot>
        ${
          targetsNewWindow(this)
            ? html`<span class="sbb-link__opens-in-new-window">
                . ${i18nTargetOpensInNewWindow[this._language.current]}
              </span>`
            : nothing
        }
      </div>
    `;

```
